### DISCUSSION TITLE
autoremove and mactex seems to not work

### DISCUSSION THREAD

**Discussion by Mazorius** [Homebrew/discussions#1810] [Everyday usage] [answered]
Hi,
I tried to clean up my macOS with the help of `brew autoremove` unfortunately this runs into issues:

``` bash
$ brew autoremove
==> Uninstalling 9 unneeded formulae:
fontconfig
freetype
ghostscript
jbig2dec
jpeg
libidn
libtiff
little-cms2
openjpeg
Error: Refusing to uninstall /usr/local/Cellar/freetype/2.10.4, /usr/local/Cellar/fontconfig/2.13.1, /usr/local/Cellar/jbig2dec/0.19, /usr/local/Cellar/jpeg/9d, /usr/local/Cellar/libidn/1.37, /usr/local/Cellar/libtiff/4.3.0, /usr/local/Cellar/little-cms2/2.12, /usr/local/Cellar/openjpeg/2.4.0 and /usr/local/Cellar/ghostscript/9.54.0
because they are required by mactex, which is currently installed.
You can override this and force removal with:
  brew uninstall --ignore-dependencies 
```

If I run `brew abv mactex` no dependencies are declared. Is this an issue in the cask formulae or is it an issue with autoremove?
Want to clarify this before I open an bug report :-) 

Thanks in advance,

**Comment by nandahkrishna** [comment, marked as answer]
I think there's already an issue (or issues) open for this: see Homebrew/brew#11417 and Homebrew/brew#11501. It seems like dependencies aren't shown when using `brew info` with casks, so you could use:
```
➜ brew deps mactex
fontconfig
freetype
ghostscript
jbig2dec
jpeg
libidn
libpng
libtiff
little-cms2
openjpeg
```
As mentioned in those issues, PRs to improve the behaviour are welcome!

**Reply by Mazorius** [reply]
Thanks @nandahkrishna !
